conda_forge_tick/pypi_name_mapping.py

```diff
@@ -264,7 +264,7 @@ def score(pkg_name):
 
     pkgs = list(gx.graph["outputs_lut"])
     ranked_list = list(sorted(pkgs, key=score))
-    with open(pathlib.Path(cf_graph) / "ranked_hubs_authorities.json", "w") as f:
```

Comment:
Are we concerned that removing this effective type cast will break things if we have been feeding strings to this function by accident?

Reply:
the only place this function is called is from the `main` function a few dozen lines down. That main function gets called from `mappings.py` but that module seems entirely superfluous and so can just be removed and replaced with a direct call to the function it's wrapping